python/ray/experimental/placement_group.py

```diff
@@ -71,21 +88,22 @@ def _get_none_zero_resource(self, bundle: List[Dict]):
 
 def placement_group(bundles: List[Dict[str, float]],
                     strategy: str = "PACK",
-                    name: str = "unnamed_group"):
-    """
-    Create a placement group.
-
-    This method is the api to create placement group.
+                    name: str = "unnamed_group") -> PlacementGroup:
+    """Asynchronously create a placement group.
 
     Args:
-        bundles: A list of bundles which represent the resources needed.
-        strategy: The strategy to create the placement group.
+        bundles(List[Dict]): A list of bundles which
+            represent the resources requirements.
+        strategy(str): The strategy to create the placement group.
             PACK: Packs Bundles into as few nodes as possible.
             SPREAD: Places Bundles across distinct nodes as even as possible.
             STRICT_PACK: Packs Bundles into one node.
             STRICT_SPREAD: Packs Bundles across distinct nodes.
             The group is not allowed to span multiple nodes.
-        name: The name of the placement group.
+        name(str): The name of the placement group.
+
+    Return:
+        Placement group object.
```

Comment:
you can actually do something like :obj:`PlacementGroup` i think and it will hyperlink correctly